chore(docs): Update overview.mdx (#1936)
hashicorp/terraform-cdk: website/docs/cdktf/api-reference/overview.mdx

---

> page_title: API Reference

Author: ```suggestion
page_title: API Reference - CDKTF Core Library
```

> page_title: API Reference - CDKTF Core Library

Author: I was thinking maybe we should be specific that this is for the core CDK library in the title and not providers, etc. Do you think this makes sense to do, or you think it's confusing?

Reviewer: Sounds good 👍

description: >-

>   Use the CDKTF core library with generated provider bindings to write your CDKTF program. Get API reference in TypeScript, Python, Java, CSharp, and Go.

Author: ```suggestion
  Use the CDKTF core library with generated provider bindings to write your CDKTF program. It is available in TypeScript, Python, Java, CSharp, and Go.
```
